script/cisetup

```diff
@@ -1,15 +0,0 @@
-#!/bin/sh
-
-# script/cisetup: Prepare CI environment for testing purposes.
-
-set -e
-
-cd "$(dirname "$0")/.."
-
-mkdir -p "$GOPATH/bin" "$GOPATH/src"
```

Comment:
I think you still need to have this setup otherwise tests will fail on ci

Reply:
We're not running tests for public repo's on our private CI infrastructure anymore, so once I get around to setting up CI for this project again, we probably won't be able to use this setup